guides/nestor/ISPsetup.md

```diff
@@ -0,0 +1,316 @@
+---
+Layout: Post
+Title: ISP setup ubuntu
+Author: Nestor N Torres
+---
+
+### Contact
+- Slack: @nanjuan on wcscusf.slack.com
+- [LinkedIn](https://www.linkedin.com/in/nestor-n-torres-737172a5/)
+- Email: nestor@nntorres.com
+
+### Table
+1. [Prerequisites](#preinfo)
+2. [Summary](#Summary)
+3. [VirtualBox VM Setup](#First)
+4. [Ubuntu Server installation](#Second)
+5. [Ubuntu Server Configuration](#Third)
+
+## Prerequisites <a id="preinfo"></a>
+1. If you have not setup your network adapter look here https://goo.gl/vs4cHC.
+2. Just copy the Prerequisites of peoples guides before yours here.
+3. Have your [virtual environment](link-to-guide) configured
+4. Have the [ISP](link-to-guide) gateway running.
+5. Have [pfSense](link-to-guide) running.
+
+## Summary <a id="Summary"></a>
+This tutorial will show you how to setup the ISP server you will need for your lab.
+
+<br>
+
+| Network adapters | Description                             |
+|------------------|---------------------------------        |
+| Adapter 1        | Leave it to NAT                         |
+| Adapter 2        | Change to Host-only (vboxnet2)          |
+
+##### Supporting detail 
+The logical ideal of this tutorial is for you to be able to setup the ISP server for your lab. The ISP lab is the one that allow you to connect your other lab to the internet think at it as your internet provider at home.  
+
+## VirtualBox VM Setup <a id="First"></a>
+
+
+1. Go to [Ubuntu](https://www.ubuntu.com/download/server) and download the LTS server version. 
+
+- Install Ubuntu Server in Virtual Machine
+- On Virtual Box go to top left corner to New to create the virtual box.
```

Comment:
It's not needed but I think the guide would be a lot better if all the button and commands were code comments. Like: On Virtual Box go to top left corner to `New` to create the virtual box. This will show the inexperienced people that they are looking for something called `New`. If any other line has that I would also suggest adding code comments.